### PR TITLE
[DOCS-9128] NDM restructure

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
This PR restructures some of the NDM content, noteably changing the "Getting Started" page to "Setup" and nesting some new pages underneath (4th level Nav).

Creates new Glossary page.

Additionally new content has been pulled and added to the Troubleshooting page.

Lastly, it appears that there are several “net new” pages, however, much of the content (new troubleshooting page, ping page, SD-WAN page, etc), have been pulled out of the Getting Started page and branched to new subpages underneath (now called Setup) 

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/network_monitoring/devices/troubleshooting.md [hunk 1]
```diff
@@ -12,91 +12,165 @@ further_reading:
 
 Use the information below for troubleshooting Datadog Network Device Monitoring. If you need additional help, contact [Datadog support][1].
 
-## Terminology
+### Device not visible in Datadog
+
+The following assumes you are running Datadog Agent version `7.59.0+`.
```
Comment (leeavital) [inline content/en/network_monitoring/devices/troubleshooting.md:17]: Per discussion on slack, there are some nicer error messages in 7.61+

```suggestion
The following assumes you are running Datadog Agent version `7.61.0+`.
```

Comment (aliciascott) [reply content/en/network_monitoring/devices/troubleshooting.md:17]: Thanks @leeavital ! Per 2nd discussion, we'll keep as is for now and iterate on this and the Autodiscovery section once 7.61 is out.

File: content/en/network_monitoring/devices/troubleshooting.md [hunk 2]
```diff
@@ -12,91 +12,165 @@ further_reading:
...
+   ```
+   Error: check device reachable: failed: error reading from socket: read udp 127.0.0.1:46068->1.2.3.4:161
+   ```
+   
+   **Solution**:
+   - Log into your device and ensure that SNMP is enabled and exposed on port 161.
+   - Ensure you collector firewall allows egress:
 
-#### What SNMP versions does Datadog support?
+   **Linux only**:
```
Comment (cswatt) [inline content/en/network_monitoring/devices/troubleshooting.md:76]: what parent section does this "Linux only" section belong to?  is it related to "Unreachable or misconfigured device" or is it its own issue?

in fact, I think this whole section could use more hierarchy—each solution should be a sub-section of its relevant issue

File: content/en/network_monitoring/devices/troubleshooting.md [hunk 3]
```diff
@@ -12,91 +12,165 @@ further_reading:
...
+   **Unreachable or misconfigured device**:
 
-## FAQ
+   ```
+   Error: check device reachable: failed: error reading from socket: read udp 127.0.0.1:46068->1.2.3.4:161
+   ```
+   
+   **Solution**:
+   - Log into your device and ensure that SNMP is enabled and exposed on port 161.
+   - Ensure you collector firewall allows egress:
```
Comment (cswatt) [inline content/en/network_monitoring/devices/troubleshooting.md:74]:
```suggestion
   - Ensure your collector firewall allows egress:
```

File: content/en/network_monitoring/devices/troubleshooting.md [hunk 4]
```diff
@@ -12,91 +12,165 @@ further_reading:
...
+   **Solution**: 
+   Ensure the following configuration parameters are correct: `user`, `authKey`, `authProtocol`, `privKey`, `privProtocol`.
 
-If you send a feature request, Datadog support needs a `snmpwalk` from the requested device. Run the following and send the output:
+### Traps not being received for devices
 
-```
-snmpwalk -O bentU -v 2c -c <COMMUNITY_STRING> <IP_ADDRESS>:<PORT> 1.3.6
-```
+1. Check the Datadog `agent.log` file to ensure that you aren't unable to bind to the traps port. If this is the case, the following error is present:
```
Comment (cswatt) [inline content/en/network_monitoring/devices/troubleshooting.md:109]:
```suggestion
Check the Datadog `agent.log` file to ensure that you can bind to the traps port. The following error indicates that you are unable to bind to the traps port:
```
this shouldn't be in the ordered list—the ordered list should just be the "solution" steps
also clearing up some unnecessarily convoluted language

File: content/en/network_monitoring/devices/troubleshooting.md [hunk 5]
```diff
@@ -12,91 +12,165 @@ further_reading:
...
+   ```
 
-- Contact [Datadog support][1] to put in a request to support your specific model.
-- Extend your profiles to support additional `sysobjectid` values. 
-    For example, if you want to monitor another type of Cisco CSR, you can modify the ISR profile directly to list another `sysobjectid` like this: 
-    
-    ```
-		snmpwalk -v 2c -c [community string] [ip address] 1.3.6.1.2.1.1.2
-    ```
+2. Navigate to the troubleshooting dashboard in NDM:
```
Comment (cswatt) [inline content/en/network_monitoring/devices/troubleshooting.md:122]: Wait, when do I move on to step 2? There was a solution in step 1, what if that solved my problem? Should I keep going?

File: content/en/network_monitoring/devices/troubleshooting.md [hunk 6]
```diff
@@ -12,91 +12,165 @@ further_reading:
...
+
+   {{< img src="/network_device_monitoring/troubleshooting/ndm_troubleshooting_dashboard.png" alt="The Network Device Monitoring page showing the Dashboard dropdown with the NDM Troubleshooting dashboard highlighed." style="width:80%;" >}}
 
-**Note**: If you do not know the `sysobjectid` of your device, try doing an Internet search or run a `snmpwalk` on a host that can reach out to your device. Use the output to list the profile to match against. 
+3. Scroll down to the Traps widget and observe the **Traps incorrectly formatted** graph. If this is non-zero it likely means that the authentication on the NDM collector and the device do not match. 
 
-#### How do I extract information about devices and interfaces of my network devices?
+   {{< img src="/network_device_monitoring/troubleshooting/ndm_traps_dashboard.png" alt="The NDM troubleshooting dashboard showing the Traps widget section." style="width:100%;" >}}
+   
+   **Solution**:
```
Comment (cswatt) [inline content/en/network_monitoring/devices/troubleshooting.md:130]: the problem/solution flow here just seems unintuitive

**Review by cswatt — CHANGES_REQUESTED** [review]
This looks good overall! I added a few suggestions, including some that are a little more in-depth. If you want to get this PR merged first before tackling the bigger ones, let me know and I'll approve whenever!

File: content/en/network_monitoring/devices/ping.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+When configured, the SNMP check can also send ICMP pings to your devices. This can be configured for individual as well as Autodiscovered devices.
+
+### Setup
+
+1. Install or upgrade the Datadog Agent to v7.52+. For platform specific instructions, see the [Datadog Agent][1] documentation.
+
+2. Edit the `snmp.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][2] for individual devices, or the [`datadog.yaml`][3] Agent configuration file for Autodiscovery. See the [sample snmp.d/conf.yaml][4] for all available configuration options.
+
+3. **Linux Only**: If you're receiving errors when running ping, you may need to configure the integration to send pings using a raw socket. This requires elevated privileges and is done using the Agent's system-probe. See the [linux.use_raw_socket][5] `system-probe` configuration below.
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/network_monitoring/devices/ping.md:22]: yes good catch thanks!

File: content/en/network_monitoring/devices/ping.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+            - 'key2:val2'
+          ping:
+            enabled: true            # (default false) enable the ping check
+            linux:                   # (optional) Linux specific configuration
+              use_raw_socket: true   # (optional, default false) send pings using a raw socket (see step 3 above)
+    ```
+
+{{% /tab %}}
+
+{{% tab "Autodiscovery" %}}
```
[the remark this answers is not in the document]
Comment (aliciascott) [reply content/en/network_monitoring/devices/ping.md:49]: ahh yeah that's a great callout and can definitely iterate on that, I also fixed the yaml formatting on this page as it was too far indented. The structure was fine it just read weird.

File: content/en/network_monitoring/devices/troubleshooting.md (hunk 2 above)
Comment (aliciascott) [reply content/en/network_monitoring/devices/troubleshooting.md:76]: it's related to the Unreachable section but yes.. Agree, let me rework this a bit!

File: content/en/network_monitoring/devices/troubleshooting.md (hunk 5 above)
Comment (aliciascott) [reply content/en/network_monitoring/devices/troubleshooting.md:122]: ahh yes that is confusing, will fix !

File: content/en/network_monitoring/devices/troubleshooting.md (hunk 6 above)
Comment (aliciascott) [reply content/en/network_monitoring/devices/troubleshooting.md]: Yes you're right! I reworked this.

**Review by cswatt — APPROVED** [review]

**Review by DavidWeid — APPROVED** [review]

**Pull request merged**